frontend_components/src/encord_active_components/frontend/src/components/explorer/api.ts

```diff
@@ -42,6 +42,7 @@ export const GroupedTagsSchema = z.object({
 export const ItemSchema = z.object({
   id: z.string(),
   url: z.string(),
+  video_timestamp: z.number().nullish(),
```

Comment:
[nitpick] 💅

should probably be consistent with the rest of the fields

```suggestion
  videoTimestamp: z.number().nullish(),
```